.github/workflows/build-test.yml

```diff
@@ -178,13 +192,15 @@ jobs:
       - id: approve-and-merge-dependabot
         name: Approve and merge dependabot PR
         if: github.actor == 'dependabot[bot]' && (steps.metadata.outputs.update-type == 'version-update:semver-patch' || steps.metadata.outputs.update-type == 'version-update:semver-minor')
-        run: gh pr review ${{ github.event.pull_request.number }} --approve -b "bors merge"
+        run: gh pr merge --auto --merge "$PR_URL"
```

Comment:
❌ This won't work:

> This enables the pull request to be merged when any tests and approvals required by the branch protection rules are successfully met

 You removed the approval, so when you enable auto-merge it will only start the merge once someone gives approval.